addon/server.js

```diff
@@ -199,7 +200,7 @@ export default class Server {
 
     // Create a collection for each factory
     _keys(factoryMap).forEach(type => {
-      let collectionName = this.schema ? pluralize(camelize(type)) : pluralize(type);
+      let collectionName = this.schema ? toDbCollectionName(type) : pluralize(type);
```

Comment:
i think these are the same now? originally schema collections were singularized and db plural, but not anymore